google-cloud-spanner/src/main/java/com/google/cloud/spanner/Session.java

```diff
@@ -28,10 +28,10 @@
  * read-write/write-only transactions, create multiple sessions. Note that standalone reads and
  * queries use a transaction internally, and count toward the one transaction limit.
  *
- * <p>Cloud Spanner limits the number of sessions that can exist at any given time; thus, it is a
- * good idea to delete idle and/or unneeded sessions. Aside from explicit deletes, Cloud Spanner can
- * delete sessions for which no operations are sent for more than an hour, or due to internal
- * errors. If a session is deleted, requests to it return {@link ErrorCode#NOT_FOUND}.
+ * <p>It is a good idea to delete idle and/or unneeded sessions. Aside from explicit deletes,
+ * Cloud Spanner can delete sessions for which no operations are sent for more than an hour,
+ * or due to internal errors. If a session is deleted, requests to it return
+ * {@link ErrorCode#NOT_FOUND}. 
```

Comment:
@rbrtwng : Please remove the whitespace at the end of this line, its failing one of the required presubmits for linting.